src/test/groovy/gex/serling/binding/UtilSpec.groovy

```diff
@@ -1,9 +1,14 @@
 package gex.serling.binding
 
-import gex.serling.binding.domain.Enemy
+import gex.serling.binding.domain.Enemy as DomainEnemy
```

Comment:
It looks weird here, but i have reviewed the characters and they are simple spaces